sap-service-operator/src/main/java/com/sap/cloud/environment/servicebinding/LayeredParsingStrategy.java

```diff
@@ -4,17 +4,36 @@
 
 package com.sap.cloud.environment.servicebinding;
 
+import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
+
+import javax.annotation.Nonnull;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Optional;
 
-import javax.annotation.Nonnull;
-
-import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
-
+/**
+ * Represents a method object to parse a {@link ServiceBinding} from a specific layered file structure (see
```

Comment:
```suggestion
 * Represents a strategy to read a {@link ServiceBinding} from a specific layered file structure (see
```

Reply:
At least for me it's not really clear what a layered structure is in this context

Reply:
That's why I linked to the `SapServiceOperatorLayeredServiceBindingAccessor`, which explains how such a structure is expected to look like in the class Javadoc.
Don't you think this should be enough?